lib/ProductOpener/Food.pm

```diff
@@ -726,37 +726,38 @@ sub mmoll_to_unit ($value, $unit) {
 			'sodium', '!carbohydrates',
 			'-fiber', '--soluble-fiber-',
 			'--insoluble-fiber-', '-sugars',
-			'-added-sugars', '--sucrose-',
+			'--added-sugars', '--sucrose-',
```

Comment:
`--added-sugars`: as I understand it will be displayed as a sub-category of sugars, isn't it? (Which is good I think.)

Does it have an impact on its name in the CSV export?

Reply:
yes, that makes added sugars a sub category of sugars (as displayed in USDA labels)